File: Beecrowd/1094/1094.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1094
tests = int(input())
total = 0; 
rabbit = 0; rat = 0; frog = 0
for i in range(tests):
    amount, animal = input().split()
    amount = int(amount)
    total += amount
    if animal == 'C':
        rabbit += amount
    elif animal == 'R':
        rat += amount
    elif animal == 'S':
        frog += amount

rabbit_percentage = (rabbit / total) * 100
rat_percentage = (rat / total) * 100
frog_percentage = (frog / total) * 100

print(f'Total: {total} cobaias')
print(f'Total de coelhos: {rabbit}')
print(f'Total de ratos: {rat}')
print(f'Total de sapos: {frog}')
print(f'Percentual de coelhos: {rabbit_percentage:.2f} %')
print(f'Percentual de ratos: {rat_percentage:.2f} %')
print(f'Percentual de sapos: {frog_percentage:.2f} %')

File: Beecrowd/1094/1094.c
/* Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1094 */
#include <stdio.h>

int main(){
    int tests, amount, total = 0;
    char animal, p = '%';
    int rabbit = 0, rat = 0, frog = 0;
    scanf("%d", &tests);
    for(int i = 0; i < tests; i++){
        scanf("%d %c", &amount, &animal);
        total += amount;
        if(animal == 'C'){
            rabbit += amount;
        }else if(animal == 'R'){
            rat += amount;
        }else if(animal == 'S'){
            frog += amount;
        }
    }
    double rabbit_percentage = (rabbit / (total*1.0)) * 100.00;
    double rat_percentage = (rat / (total*1.0)) * 100.00;
    double frog_percentage = (frog / (total*1.0)) * 100.00;
    
    printf("Total: %d cobaias\n", total);
    printf("Total de coelhos: %d\n", rabbit);
    printf("Total de ratos: %d\n", rat);
    printf("Total de sapos: %d\n", frog);
    printf("Percentual de coelhos: %.2lf %c\n", rabbit_percentage, p);
    printf("Percentual de ratos: %.2lf %c\n", rat_percentage, p);
    printf("Percentual de sapos: %.2lf %c\n", frog_percentage, p);
    
    return 0;
}

